Pod/Classes/PINAnimatedImage.h

```diff
@@ -0,0 +1,65 @@
+//
+//  PINAnimatedImage.h
+//  Pods
+//
+//  Created by Garrett Moon on 3/18/16.
+//
+//
+
+#import <Foundation/Foundation.h>
+
+#import "PINRemoteImageMacros.h"
+
+#define PINAnimatedImageDebug  0
+
+extern NSString *kPINAnimatedImageErrorDomain;
+
+typedef NS_ENUM(NSUInteger, PINAnimatedImageError) {
+  PINAnimatedImageErrorNoError = 0,
+  PINAnimatedImageErrorFileCreationError,
+  PINAnimatedImageErrorFileHandleError,
+  PINAnimatedImageErrorImageFrameError,
+  PINAnimatedImageErrorMappingError,
+};
+
+typedef NS_ENUM(NSUInteger, PINAnimatedImageStatus) {
+  PINAnimatedImageStatusUnprocessed = 0,
+  PINAnimatedImageStatusInfoProcessed,
+  PINAnimatedImageStatusFirstFileProcessed,
```

Comment:
Could you provide a comment on what this means? You just explained it to me and it makes sense, but I kind of thought that you meant first frame processed until i understood what was going on.